app/javascript/src/components/Dashboard/Notes/NoteList/Note.jsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+
+import { Clock } from "@bigbinary/neeto-icons";
+import { Avatar, Tooltip } from "@bigbinary/neetoui";
+
+import DropDownNote from "./DropDownNote";
+
+const Note = ({
+  note,
+  setSelectedDeleteNoteIds,
+  setSelectedNote,
+  setShowEditNote,
+  setShowDeleteAlert,
+}) => {
+  const handleEditClick = () => {
+    setSelectedNote(note);
+    setShowEditNote(true);
+  };
+  const handleDeleteClick = () => {
+    setSelectedDeleteNoteIds([note.id]);
+    setShowDeleteAlert(true);
+  };
+  return (
```

Comment:
Go through the slack thread and fix the mistake in this block.

Reply:
Done. Changes will be reflected in the next PR.